builder/ebs/step_prepare_fast_launch_template.go

```diff
@@ -49,6 +54,14 @@ func (s *stepPrepareFastLaunchTemplate) Run(ctx context.Context, state multistep
 	for _, templateSpec := range s.RegionTemplates {
 		region := templateSpec.Region
 
+		if templateSpec.EnableFalseLaunch == config.TriFalse {
+			log.Printf("[INFO] fast-launch explicitly disabled for region %q", region)
+			templateIDsByRegion[region] = TemplateSpec{
+				Enabled: false,
```

Comment:
I'm not to clear on how this works by default but from what I gather (looking at the issue) fast launch seems to be enabled by default on the region. Is that correct? 

If so, does the user need any particular permissions to disable on the region?

Reply:
No permissions necessary, we enable fast-launch for each region in a later step, so if you don't want it on a particular region, we don't request it be enabled